src/strategyHelpers/blueGreen/blueGreenHelper.test.ts

```diff
@@ -0,0 +1,80 @@
+import {createWorkloadsWithLabel, deleteWorkloadsAndServicesWithLabel, getManifestObjects, getNewBlueGreenObject, GREEN_LABEL_VALUE, isServiceRouted, NONE_LABEL_VALUE} from './blueGreenHelper'
+import * as bgHelper from './blueGreenHelper'
+import { Kubectl } from '../../types/kubectl'
+import * as fileHelper from '../../utilities/fileUtils'
+
+jest.mock('../../types/kubectl')
+var testObjects
```

Comment:
moving testObjects inside one level to the describe and using let would be cleaner. Var should just be avoided as much as possible